src/models/commons.ts

```diff
@@ -1,9 +1,25 @@
 import {PinataCredentials} from "./pinata";
 
-export interface RaribleSDKConfiguration {
-    pinataCredentials: PinataCredentials;
+export interface AssetType {
+    assetClass: number,
+    /**
+    * This is generic data because the format of data is varies by asset class.
+    * For example, for an asset class of ERC-20, the data holds the address of the token.
+    * For ERC-721 data holds smart contract address and tokenId.
+    **/
+    data: any,
+}
+
+export interface Asset {
+    amount: number,
+    assetType: AssetType,
+}
```

Comment:
Is it possible to list all the possible asset types? if it's possible we could create interfaces to force typing depending on the asset type

Reply:
I believe I've addressed this in fb91ffa